Wikipedia/assets/bundle.js

```diff
@@ -381,8 +381,10 @@ global.scrollToFragment = scrollToFragment;
 (function (global){
 var utilities = require("./utilities");
 
```

Comment:
Now we can control exactly which level of headers we are going to overlay with native ones.